plugins/assimp/module/vtkF3DAssimpImporter.cxx

```diff
@@ -88,8 +88,6 @@ class vtkF3DAssimpImporter::vtkInternals
 
       if (nbLights > 0)
       {
-        renderer->AutomaticLightCreationOff();
```

Comment:
I remembered doing that on purpose. The idea is to remove VTK lights when there is at least one light provided in the file.

Reply:
not needed:

https://github.com/Kitware/VTK/blob/master/Rendering/OpenGL2/vtkOpenGLRenderer.cxx#L141

Reply:
(and light test would fail if that were not true.)

Reply:
(and the renderer parameters should not be changed by the importer anyway)